crates/ruma-common/tests/api/ui/deprecated-without-added.rs

```diff
@@ -0,0 +1,15 @@
+use ruma_common::{api::Metadata, metadata};
+
+const _: Metadata = metadata! {
+    description: "This will fail.",
+    method: GET,
+    name: "invalid_versions",
+    unstable => "/a/path",
+    rate_limited: false,
+    authentication: None,
+    history: {
+        1.1 => deprecated,
+    }
```

Comment:
Er... this is not correct, the `unstable` path should be in `history`, and this error should instead come from the `const fn` raising a panic about the absence of a version that added it.